uxarray/conventions/ugrid.py

```diff
@@ -0,0 +1,323 @@
+from uxarray.constants import INT_FILL_VALUE, INT_DTYPE
+
+CONVENTIONS_ATTR = "UGRID-v1.0"
+
+# minimum grid topology, additions are made depending on what is present in a grid
+BASE_GRID_TOPOLOGY_ATTRS = {
+    "cf_role": "mesh_topology",
+    "topology_dimension": 2,
+    # dimensions
+    "face_dimension": "n_face",
+    "node_dimension": "n_node",
+    # coordinates
+    "node_coordinates": "node_lon node_lat",
+    # connectivity
+    "face_node_connectivity": "face_node_connectivity",
+}
+
+DIM_NAMES = [
+    "n_node",
+    "n_edge",
+    "n_face",
+    "n_max_face_nodes",
+    "n_max_face_edges",
+    "n_max_face_faces",
+    "n_max_edge_nodes",
+    "n_max_edge_edges",
+    "n_max_edge_faces",
+    "n_max_node_faces",
+    "n_max_node_edges",
+    "two",
+]
+
+NODE_DIM = "n_node"
+EDGE_DIM = "n_edge"
+FACE_DIM = "n_face"
+N_MAX_FACE_NODES_DIM = "n_max_face_nodes"
+
+NODE_COORDINATES = ["node_lon", "node_lat"]
```

Comment:
Should we also consider the Cartesian Coordinates here or for future use? Specifically, should we name it as `SPHERICAL_NODE_COORDINATES` instead of just `SPHERICAL_NODE_COORDINATES`?

Reply:
From the UGRID conventions

> The attribute node_coordinates points to the auxiliary coordinate variables representing the node locations (latitude, longitude, or other spatial coordinates, and optional elevation or other coordinates).

We should be able to set `NODE_COORDINATES` to what you outlined. I'll give this a try.

Reply:
The same can be done for `EDGE_COORDINATES` and `FACE_COORDINATES`